rust-mode.el

```diff
@@ -374,6 +381,51 @@
              ("fn" . font-lock-function-name-face)
              ("static" . font-lock-constant-face)))))
 
+
+(defun rust-extend-region-nested-comment ()
+  ;; Extend the region given by `font-lock-beg' and `font-lock-end' to
```

Comment:
I agree they should be docstrings.

I opened an issue to run checkdoc as part of the test suite, I think it's a must-have for any elisp package. #70